fixes #14855 - hide template type dropdown when snippet checked

domcleal opened the pull request.

domcleal committed:
fixes #14855 - hide template type dropdown when snippet checked

dLobatog commented on app/assets/javascripts/editor.js:
```diff
@@ -103,7 +103,7 @@ function editor_file_source(evt){
 
 function snippet_changed(item){
-  var checked = !!$(item).attr('checked');
+  var checked = $(item).is(':checked')
```
Nitpick, missing semicolon

dLobatog commented:
@domcleal :+1: thanks! I don't think the test failures are related - one of them is fixed on your other PR. Can you write an integration test to ensure we don't get the same regression again?

domcleal commented:
Sure, I've added JS integration tests for snippets and templates to check the field hides/shows properly and that editing templates works. I've fixed the missing semicolon - thanks!

dLobatog commented:
:+1: cool stuff, I'll merge if Mr Jenkins gives the OK

dLobatog commented:
Some process got killed in jenkins, other than that is green. Merging !

http://ci.theforeman.org/job/test_develop_pr_core/9009/database=sqlite3,label=fast,ruby=2.0.0/console

dLobatog commented:
Merged as deb71b8733b431d9637cbcdc889f994b930989bc, thanks @domcleal!
